### PR TITLE
Auto-generated Entity Types

### PR DESCRIPTION
**Pull request by maxheckel**
Creates auto-generation of types based on the json schema from the data model.  Introduces a new package `integration-sdk-entities` to contain these types so we don't run into any conflicting names for common entity names (i.e. `Entity`)

These will eventually be imported into the `integration-sdk-core` to allow `graph` libraries to use these instead of raw javascript objects when building integrations, hopefully significantly reducing the developer cost to know and validate that their code is working.

### CODE REVIEW

File: packages/integration-sdk-entities/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@jupiterone/integration-sdk-entities",
+  "version": "8.35.0",
+  "description": "The SDK for developing JupiterOne integrations",
```
Comment (zemberdotnet) [inline packages/integration-sdk-entities/package.json:4]:
```suggestion
  "description": "Generated types for the JupiterOne data-model",
```

File: packages/integration-sdk-entities/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# @jupiterone/integration-sdk-entities
+
+This package contains all entities generated from J1's data model to be used
+throughout J1 systems.
```
Comment (zemberdotnet) [inline packages/integration-sdk-entities/README.md:4]: I'm okay with it being entities specific, but the `data-model` technically defines the schema for all graph objects including relationships.

File: packages/integration-sdk-entities/tools/copy-schemas/_schemas/.gitkeep
Comment (zemberdotnet) [inline packages/integration-sdk-entities/tools/copy-schemas/_schemas/.gitkeep:1]: I've actually never used `.gitkeep` before. Is this just here so the directory is created?

File: packages/integration-sdk-entities/tools/create-barrel/index.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    const filePath = path.join(dir, file);
+    const typeBlob = await fs.readFile(filePath);
+    const typeString = typeBlob.toString();
+    const empty = `/* eslint-disable */
+`;
+    if (typeString !== empty) {
+      fileNames.push(file);
+    }
+  }
+
```
Comment (zemberdotnet) [inline packages/integration-sdk-entities/tools/create-barrel/index.js:42]: Adding a comment on what this file does may be helpful. It seems pretty specific to the way we are processing things, but I can't quite make out what the purpose is.

**Review by zemberdotnet — APPROVED** [review]

File: packages/integration-sdk-entities/README.md (hunk 2 above)
Comment (maxheckel) [reply packages/integration-sdk-entities/README.md:4]: Made it more explicit

**Review by zemberdotnet — APPROVED** [review]

**Pull request merged**